### PR TITLE
stolostron/backlog#19025 add imagestorage

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_infra_env.adoc [hunk 1]
```diff
@@ -75,6 +81,8 @@ Replace `db_volume_size` with the volume size for the `databaseStorage` field, f
 +
 Replace `fs_volume_size` with the size of the volume for the `filesystemStorage` field, for example `50MB`. This value specifies how much storage is allocated for storing logs, manifests, and `kubeconfig` files for the clusters. You might need to use a higher value if there are many clusters. 
 +
+Replace `img_volume_size` with the size of the volume for the `imageStorage` field, for example `2GB`. This value specifies how much storage is allocated for storing images for the clusters. You might need to use a higher value if there are many clusters.
```
Comment (filanov) [inline clusters/create_infra_env.adoc]: We need 1GB for each RCHOS so i suggest to set it here to 10 and explain the user the reason behind it

**Review by filanov — COMMENTED** [review]
We didn't backport it into 2.4.
It's a big change

**Comment by cadawson** [comment]
Thanks for the feedback @filanov , and for letting me know that it was not backported to 2.4. We will leave it only in 2.5, then. I added an update, based on your comments. Please review and let me know if it looks ok.

File: clusters/create_infra_env.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/create_infra_env.adoc]: Made an update for this.

**Comment by filanov** [comment]
LGTM

**Comment by cadawson** [comment]
Thanks @filanov !

**Pull request merged**